.ci.yaml

```diff
@@ -4681,6 +4681,57 @@ targets:
       tags: >
         ["framework", "hostonly", "shard", "windows"]
 
+  - name: Linux flutter_packaging_stable
+    recipe: packaging_v2/packaging_v2
+    timeout: 60
+    scheduler: release
+    bringup: true
+    enabled_branches:
+      - stable
+    properties:
+      task_name: flutter_packaging_stable
+      tags: >
+        ["framework", "hostonly", "shard", "linux"]
+
+
+  - name: Mac flutter_packaging_stable
+    recipe: packaging_v2/packaging_v2
+    timeout: 60
+    scheduler: release
+    bringup: true
+    enabled_branches:
+      - stable
```

Comment:
As the configurations are identical can we just add enabled_branches to the existing beta ones and remove _beta from the name?

Reply:
Yes that makes sense. I will try and see what TESTOWNERS thinks